src/libs/TransactionUtils.js

```diff
@@ -60,13 +60,16 @@ function buildOptimisticTransaction(
         commentJSON.originalTransactionID = originalTransactionID;
     }
 
+    // For the SmartScan to run successfully, we need to pass the merchant field empty to the API
+    const defaultMerchant = _.isEmpty(receipt) ? CONST.TRANSACTION.DEFAULT_MERCHANT : '';
+
     return {
         transactionID,
         amount,
         currency,
         reportID,
         comment: commentJSON,
-        merchant: merchant || CONST.TRANSACTION.DEFAULT_MERCHANT,
+        merchant: merchant || defaultMerchant,
```

Comment:
This change was redundant. It's just optimistic data, not affect api call.
As merchant is set to empty optimistically, it caused [description briefly showing](https://github.com/Expensify/App/issues/28380) before api response.